### PR TITLE
test not working for breadthFirst

### PR DESCRIPTION
**Pull request by tcoots88**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/test/java/code401challenges/tree/TreeTest.java [hunk 1]
```diff
@@ -69,4 +69,15 @@ public void testForPostOrderMultiNode(){
...
+
+    @Test
+    public void testForBreadthFirst(){
+        Tree<Integer> testTree = new Tree<>(12);
+        testTree.root.setLeft(new Node<>(13));
+        testTree.root.getLeft().setLeft(new Node<>(18));
+        testTree.root.setRight(new Node<>(14));
+        testTree.root.getRight().setLeft(new Node<>(16));
+        testTree.root.getRight().setRight(new Node<>(17));
+        System.out.println("testTree.breadthFirst(testTree.root.getData()) = " + testTree.breadthFirst(testTree.root.getData()));;
```
Comment (JamesDansie) [inline code401challenges/src/test/java/code401challenges/tree/TreeTest.java]: To run it you'll need;
`        testTree.breadthFirst(testTree.root);
`

File: code401challenges/src/main/java/code401challenges/tree/Tree.java [hunk 2]
```diff
@@ -73,4 +76,21 @@ public void setRoot(Generic data) {
...
+
+// breadth-first code challenge
+
+    public void breadthFirst(Node root) {
+        Queue<Node> queue = new LinkedList<Node>();
+        if (root == null)
+            return;
+        queue.add(root);
+        while (!queue.isEmpty()) {
+            Node n = (Node) queue.remove();
```
Comment (JamesDansie) [inline code401challenges/src/main/java/code401challenges/tree/Tree.java]:
```suggestion
            Node n = (Node) queue.remove();
            System.out.println(n.data);
```
^ will help debug
